### PR TITLE
jgrss/crs error

### PR DESCRIPTION
**Pull request by jgrss**
## What is this PR changing?
* Changes 'stac' extra to forked `stackstac` to use CRS fix
* Adds support to write `numpy` arrays as `DataArray` 'data

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -12,7 +12,6 @@ python:
         - docs
         - perf
         - ml
-        - stac
```
Comment (jgrss) [inline .readthedocs.yml:15]: I think this was causing memory issues in ReadTheDocs.

File: src/geowombat/backends/rasterio_.py [hunk 2]
```diff
@@ -203,7 +203,12 @@ def close_delayed(self, store):
         return self.close()
 
     def write(self, data: xr.DataArray, compute: bool = False) -> Delayed:
-        return da.store(da.squeeze(data.data), self, lock=True, compute=compute)
+        if isinstance(data.data, da.Array):
```
Comment (jgrss) [inline src/geowombat/backends/rasterio_.py:206]: Allows a `DataArray` to have `numpy` values + `dask` values.

File: src/geowombat/core/io.py [hunk 3]
```diff
@@ -670,19 +670,30 @@ def save(
         if dtype != 'float32':
             dtype = 'float64'
 
+    blockxsize = (
```
Comment (jgrss) [inline src/geowombat/core/io.py:673]: Necessary in case the array is `numpy`.

**Review by mmann1123 — APPROVED** [review]
Ok looks good!

**Pull request merged**